examples/n_monitoring.jl

```diff
@@ -61,14 +59,7 @@ for s in paths([State(2) for i in 1:N])
 end
 add_utilities!(diagram, "T", Y_T)
 
-generate_diagram!(diagram, positive_path_utility=true)
-
-
-model = Model()
-z = DecisionVariables(model, diagram)
-x_s = PathCompatibilityVariables(model, diagram, z, probability_cut = false)
-EV = expected_value(model, diagram, x_s)
-@objective(model, Max, EV)
+model, z, variables = generate_model(diagram, model_type="RJT")
```

Comment:
I think that we should solve one example with the path based formulation just to show how it is done. N-monitoring had the least difference between RJT and DecProg so I think this could be solved with DecProg